CHANGES.md

```diff
@@ -51,6 +55,23 @@ To be released.
 
  -  Added `IAccountDelta` interface and its default implementation
     `AccountDelta` class.  [[#3245]]
+ -  Added `Maj23` and its related classes.  [[#3249]]
+     -  Added `Maj23` class.
+     -  Added `Maj23Metadata` class.
+     -  (Libplanet.Net) Added `ConsensusMaj23Msg` class.
+ -  Added `VoteSetBits` and its related classes.  [[#3206]]
+     -  Added `VoteSetBits` class.
+     -  Added `VoteSetBitsMetadata` class.
+     -  (Libplanet.Net) Added `ConsensusVoteSetBitsMsg` class.
+ -  (Libplanet.Net) Added `VoteSet` class.  [[#3249]]
+ -  (Libplanet.Net) Added `HeightVoteSet` class.  [[#3249]]
+ -  (Libplanet.Net) Added `ConsensusVoteMsg` abstract class.  [[#3249]]
+ -  (Libplanet.Net) Added `InvalidProposalException` class.  [[#3249]]
+ -  (Libplanet.Net) Added `InvalidVoteException` class.  [[#3249]]
+ -  (Libplanet.Net) Added `InvalidMaj23Exception` class.  [[#3249]]
+ -  (Libplanet.Net) Added
+    `Gossip.PublishMessage(MessageContent, IEnumerable<BoundPeer>)` method.
+    [[#3249]]
 
 ### Behavioral changes
 
```

Comment:
I think change on `nil` vote expression have to be expressed. (`BlockHash = null` => `BlockHash = default`)